service/frontend/adminHandler.go

```diff
@@ -19,6 +19,7 @@
 // THE SOFTWARE.
 
 //go:generate mockgen -package $GOPACKAGE -source $GOFILE -destination adminHandler_mock.go -package frontend github.com/uber/cadence/service/frontend AdminHandler
+//go:generate gowrap gen -g -p . -i AdminHandler -t ./templates/accesscontrolled.tmpl -o ./access_controlled_admin_gen.go -v handler=AccessControlled
```

Comment:
I propose to separate code from generated code. 
Also, naming convention for generated files is _generated.go https://github.com/uber/cadence/pull/5574
I don't think it is set in stone, but coverage wise, I think we should be consistent

Reply:
Originally, I put them in a separate package but realized that introduced a cyclic dependency.

Reply:
It should be fine that generated files are in same folder but let's follow same suffix convention "_generated.go"

Reply:
I need to do another refactoring to remove the cyclic dependency.